### ISSUE TITLE
Make Accessible for keyboard users

### ISSUE THREAD

**Issue by Ben-mav** [OwlCarousel2/OwlCarousel2#1717]
Hi,

My team of web developers are using the Owl Carousel for a homepage and have been unable to make this accessible for keyboard users.

Can you please make this Double AA accessible for keyboard users?

That would require it stop auto-rotating  ##when tabbing through, and the dots to update as you tab through the carousel.

Cheers!

**Comment by CracyCrazz** [comment]
Hey @Ben-mav,

this is a feature we are looking forward to, but I can't give you yet a date when this will be released.

**Comment by Ben-mav** [comment]
Thanks @CracyCrazz, appreciate the response!

**Comment by tonla-azure** [comment]
@daviddeutsch We are currently using the carousel, if possible we would like to contribute to the Accessibility of the carousel. It is a critical piece of the site.

**Comment by daviddeutsch** [comment]
Some notes for things we need to keep in mind:

- The dots should be rendered as <button> elements
- The next/prev elements should be <button> elements
- The cloned hidden slides have selectable elements on them where focus is lost
- When tabbing through the slides, there are layout issues, half of two different slides show (maybe Firefox only)

**Comment by llahnoraa** [comment]
Following

**Comment by MattWilsonMD** [comment]
It's 2021... time to make this thing accessible. 
When using the tab key to navigate a page with the Owl Carousel, I have to tab thru all the "clone" items. Just give them a tabindex of -1 and we should be good to go.
